src/main/java/seedu/address/logic/Messages.java

```diff
@@ -14,7 +14,8 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is out of bounds "
+            + "from the end of the list";
```

Comment:
Not sure whether do we hv to standardise the use of elderly in Message also...

Reply:
Sure I'll do that